pinot-filesystem/src/main/java/com/linkedin/pinot/filesystem/PinotFS.java

```diff
@@ -51,10 +51,11 @@
    * it will overwrite. Will work either for moving a directory or a file.
    * @param srcUri URI of the original file
    * @param dstUri URI of the final file location
+   * @param overwrite true if we want to overwrite the dstURI, false otherwise
```

Comment:
Please clarify what happens when move is called with different combinations. What is src is a file and dst a directory, or the other way around? If they are both directories, then is the source moved _under_ the dest directory or is it moved _as_ destination directory?